lib/server/routers/qcFlags.router.js

```diff
@@ -0,0 +1,32 @@
+/**
+ *  @license
+ *  Copyright CERN and copyright holders of ALICE O2. This software is
+ *  distributed under the terms of the GNU General Public License v3 (GPL
+ *  Version 3), copied verbatim in the file "COPYING".
+ *
+ *  See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ *  In applying this license CERN does not waive the privileges and immunities
+ *  granted to it by virtue of its status as an Intergovernmental Organization
+ *  or submit itself to any jurisdiction.
+ */
+
+const { QCFlagTypesController } = require('../controllers/QCFlagTypes.controller.js');
+
+exports.QCFlagsRouter = {
+    path: '/qualityControlFlags',
```

Comment:
`qcFlags`

Why not creating a dedicated endpoint?

Reply:
It could be, by it would be like `/qcFlagTypes` instead of `/qcFlags/types`, I prefer second one, as it is, I would say natural structuring in this particular case, but if it's crucial for some reasons I can change it

Reply:
Well, it is a convention of the endpoints https://github.com/AliceO2Group/Bookkeeping/blob/main/docs/CONTRIBUTING.md#endpoint-conventions

Reply:
applied